htdocs/product/class/html.formproduct.class.php

```diff
@@ -660,7 +660,7 @@ public function selectLotDataList($htmlname = 'batch_id', $empty = 0, $fk_produc
 					if (empty($fk_entrepot) || $fk_entrepot == $arraytypes['entrepot_id']) {
 						$label = $arraytypes['entrepot_label'] . ' - ';
 						$label .= $arraytypes['batch'];
-						$out .= '<option>' . $arraytypes['batch'] . '</option>';
+						$out .= '<option value="' . $arraytypes['batch'] . '">(' . $langs->trans('Stock Total') . ': ' . $arraytypes['qty'] . ')</option>';
```

Comment:
The $arraytypes['batch'] is added into value but does not appear into visible part ?  (between <option> and </option>

Reply:
Hi @eldy, I just tried what @atm-lena did, and it seems to work properly :

Before : 
![image](https://user-images.githubusercontent.com/5585819/190112810-d37525a3-b72e-40ee-a977-198db62caf99.png)

After :
![image](https://user-images.githubusercontent.com/5585819/190113176-5f440896-78ba-4f6b-bf80-b1f2c8d6135e.png)